ansible/roles/schulcloud-server-core/templates/admin-api-server-deployment.yml.j2

```
@@ -0,0 +1,64 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: admin-api-deployment
+  namespace: {{ NAMESPACE }}
+  labels:
+    app: api-admin
+    app.kubernetes.io/part-of: schulcloud-verbund
+    app.kubernetes.io/version: {{ SCHULCLOUD_SERVER_IMAGE_TAG }}
+    app.kubernetes.io/name: api-admin
+    app.kubernetes.io/component: server
+    app.kubernetes.io/managed-by: ansible
+    git.branch: {{ SCHULCLOUD_SERVER_BRANCH_NAME }}
+    git.repo: {{ SCHULCLOUD_SERVER_REPO_NAME }}
+spec:
+  replicas: {{ TLDRAW_SERVER_REPLICAS|default("1", true) }}
```

It's not tldraw.
Missing also the settings for it at the dof_app_deploy pr
```suggestion
  replicas: {{ ADMIN_API_SERVER_REPLICAS|default("1", true) }}
```

---

ohh right

---

What values should I put in dof app deploy

---

Default values are ok to put in dof app deploy?

---

Make a Proposal for it at dof_app_deploy and then we can see.
From where and how offen it is calld what do you thing about restart?
If it only internal and only less calles then maybe one is okay